portal-kernel/src/com/liferay/portal/kernel/workflow/permission/WorkflowPermissionUtil.java

```diff
@@ -101,7 +101,7 @@ private static Boolean _hasPermission(
 				permissionChecker, workflowInstance);
 
 			if (!hasPermission && actionId.equals(ActionKeys.VIEW)) {
-				return null;
+				return false;
```

Comment:
@vngoliveira The method signature requires a Boolean Java Object,
Try to use like in line [89](https://github.com/liferay/liferay-portal/blob/e1c1485ab3210fdcd5abab6226c295bf3617a14a/portal-kernel/src/com/liferay/portal/kernel/workflow/permission/WorkflowPermissionUtil.java#L89) or line [75](https://github.com/liferay/liferay-portal/blob/e1c1485ab3210fdcd5abab6226c295bf3617a14a/portal-kernel/src/com/liferay/portal/kernel/workflow/permission/WorkflowPermissionUtil.java#L75).
Would look like this:
```suggestion
				return Boolean.FALSE;
```